modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/TypeScriptRxjsClientCodegen.java

```diff
@@ -228,4 +296,71 @@ private void addExtraReservedWords() {
         this.reservedWords.add("Middleware");
         this.reservedWords.add("AjaxResponse");
     }
+
+    class ExtendedCodegenOperation extends CodegenOperation {
+        public boolean hasHttpHeaders;
+
+        public ExtendedCodegenOperation(CodegenOperation o) {
+            super();
+
+            // Copy all fields of CodegenOperation
```

Comment:
Without having checked in the code, would it be possible to call `super(o)` instead of copying all the assignments?

Reply:
Unfortunately no.
I am no java expert but I think the reason is that there is only the default constructor.
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.5.1:compile (default-compile) on project openapi-generator: Compilation failure
[ERROR] .../openapi-generator/src/main/java/org/openapitools/codegen/languages/TypeScriptRxjsClientCodegen.java:[304,13] constructor CodegenOperation in class org.openapitools.codegen.CodegenOperation cannot be applied to given types;
[ERROR]   required: no arguments
[ERROR]   found: org.openapitools.codegen.CodegenOperation
[ERROR]   reason: actual and formal argument lists differ in length
```